react/components/AccordionFilterItem.js

```diff
@@ -2,11 +2,11 @@ import React, { Fragment } from 'react'
 import PropTypes from 'prop-types'
 import { intlShape, injectIntl } from 'react-intl'
 import classNames from 'classnames'
+import { IconCaret } from 'vtex.store-icons'
 import { Tag } from 'vtex.styleguide'
 import { useCssHandles } from 'vtex.css-handles'
```

Comment:
You're trying to use `applyModifiers`, but you're not importing it. That's what is causing the tests to fail, and this should also prevent the app from linking and building. Are you able to link this? 🤔

Reply:
@victorhmp Yes, this workspace have this app linked 🤔 
https://filter--carrefourbr.myvtex.com/celulares-e-smartphones/smartphones

Well, I updated the component. Please review it again 🙏